tools/c7n_org/c7n_org/cli.py

```diff
@@ -486,7 +486,7 @@ def run_account(account, region, policies_config, output_path,
                 log.error(
                     "Exception running policy:%s account:%s region:%s error:%s",
                     p.name, account['name'], region, e)
-                continue
+                raise SystemExit(1)
```

Comment:
not clear this is actually going to do the intended effect, c7n-org runs multi-process, this just kills the child, not the parent afaics.